src/cli.coffee

```diff
@@ -24,6 +25,11 @@ publishVersion = (str, currentVersion) ->
   return version
 
 cli = (argv, directory, packageJson) ->
+  if argv.v or argv.version
+    npubPackageJson = require path.join(__dirname, '..', 'package.json')
```

Comment:
Haha yeah, I implemented it that way first, but then I worried for no good reason at all.